### ISSUE TITLE
Failing Thermo/Symmetry Tests

### ISSUE THREAD

**Issue by mliu49** [ReactionMechanismGenerator/RMG-Py#887]
I noticed a few unit tests that were marked @work_in_progress by @rwest almost 3 years ago that seem fairly important in rmgpy.data.thermoTest.

I made some changes to have the test work properly again, and found that they are now failing due to differences in our calculations. Please see the [fixThermoTest](https://github.com/ReactionMechanismGenerator/RMG-Py/commits/fixThermoTest) branch for changes. It's possible that this was the original reason that the tests were marked @work_in_progress, and changes to RMG caused the error to shift to syntax problems.

Specifically, the S298 values for cyclopentadiene and naphthalene are different from the benchmark values, and their symmetry numbers are also wrong. For S298, I'm not sure if the benchmarks need to be updated, or if we're just getting it wrong.

**Comment by rwest** [comment]
Just noticed my commit message from that commit:
```
Add @work_in_progress annotations to some failing thermoTests

We mustn't get lazy and forget to fix these...
```

Erm..., hahaha?

**Comment by goldmanm** [comment]
I have looked at the commented out symmetry tests. I think most of them exist there since we currently don't have an algorithm that correctly finds the symmetry. Keeping them there as a sort of 'reminder' might have been the intent...though on another level it also hides the errors. Some issues are also posted which related directly to commented out tests (#858, #844, #332, #119). I also wrote more tests for cyclics on my branch, but since I wasn't revamping the algorithm, I commented them out.

Do we have a preferred way to identify these molecules as issues in our symmetry algorithm?
